model/src/main/proto/interaction_object.proto

```diff
@@ -6,6 +6,7 @@ option java_package = "org.oppia.app.model";
 option java_multiple_files = true;
 
 // Structure for any interaction object
+// https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py
```

Comment:
Please reference a specific commit hash instead of the develop branch to create a permalink. Otherwise if the file is moved or renamed in the future this link won’t work anymore.

Reply:
done